### ISSUE TITLE
linter includes existing valid todos as errors when directly run against file path

### ISSUE THREAD

**Issue by shankarsridhar** [ember-template-lint/ember-template-lint#2287]
First of all, thanks for this wonderful addon; I find it useful(the todo is a game changer for me).

**TL;DR** - Running the linting command against a file path directly seems to incorrectly include existing valid todos as errors. This does not happen with folder paths.

I started integrating this addon in my ember app that has more than 1000 `.hbs` linting errors, and I updated them all as todos using the `--update-todo` option.

I tried both `3.15.0` and `4.0.0-beta.1` and ran into this issue with both these versions.

**Worked as expected**
I intentionally added a new error(`{{log}}`) in a file(`application.hbs` _line 41_) and ran the command to verify if only the new error shows up (while the old errors have been updated as todo), and it worked as expected.

_Command_:
```node_modules/ember-template-lint/bin/ember-template-lint.js ./app/templates/```

_Output_:
```➜  capacity git:(ember-template-lint) ✗ node_modules/ember-template-lint/bin/ember-template-lint.js ./app/templates/
app/templates/application.hbs
  41:0  error  Unexpected {{log}} usage.  no-log

✖ 1 problems (1 errors, 0 warnings)
```

**Did not work as expected**
when I ran the command directly against the file using its path, it did not work as expected: 

_Command:_
```node_modules/ember-template-lint/bin/ember-template-lint.js ./app/templates/application.hbs```

_Output:_
```
➜  capacity git:(ember-template-lint) ✗ node_modules/ember-template-lint/bin/ember-template-lint.js ./app/templates/application.hbs
./app/templates/application.hbs
  15:4  error  Do not use `action` as <button {{action ...}} />. Instead, use the `on` modifier and `fn` helper.  no-action
  21:52  error  Do not use `action` as <button {{action ...}} />. Instead, use the `on` modifier and `fn` helper.  no-action
  27:39  error  Do not use `action` as (action ...). Instead, use the `on` modifier and `fn` helper.  no-action
  1:0  error  You are using the component {{head-layout}} with curly component syntax. You should use <HeadLayout> instead. If it is actually a helper you must manually add it to the 'no-curly-component-invocation' rule configuration, e.g. `'no-curly-component-invocation': { allow: ['head-layout'] }`.  no-curly-component-invocation
  27:0  error  You are using the component {{insights/glossary-modal}} with curly component syntax. You should use <Insights::GlossaryModal> instead. If it is actually a helper you must manually add it to the 'no-curly-component-invocation' rule configuration, e.g. `'no-curly-component-invocation': { allow: ['insights/glossary-modal'] }`.  no-curly-component-invocation
  1:2  error  Ambiguous path 'head-layout' is not allowed. Use '@head-layout' if it is a named argument or 'this.head-layout' if it is a property on 'this'. If it is a helper or component that has no arguments, you must either convert it to an angle bracket invocation or manually add it to the 'no-implicit-this' rule configuration, e.g. 'no-implicit-this': { allow: ['head-layout'] }.  no-implicit-this
  14:36  error  Ambiguous path 'isShowWalkthrough' is not allowed. Use '@isShowWalkthrough' if it is a named argument or 'this.isShowWalkthrough' if it is a property on 'this'. If it is a helper or component that has no arguments, you must either convert it to an angle bracket invocation or manually add it to the 'no-implicit-this' rule configuration, e.g. 'no-implicit-this': { allow: ['isShowWalkthrough'] }.  no-implicit-this
  20:8  error  Ambiguous path 'isShowGlossary' is not allowed. Use '@isShowGlossary' if it is a named argument or 'this.isShowGlossary' if it is a property on 'this'. If it is a helper or component that has no arguments, you must either convert it to an angle bracket invocation or manually add it to the 'no-implicit-this' rule configuration, e.g. 'no-implicit-this': { allow: ['isShowGlossary'] }.  no-implicit-this
  27:70  error  Ambiguous path 'isGlossaryOpen' is not allowed. Use '@isGlossaryOpen' if it is a named argument or 'this.isGlossaryOpen' if it is a property on 'this'. If it is a helper or component that has no arguments, you must either convert it to an angle bracket invocation or manually add it to the 'no-implicit-this' rule configuration, e.g. 'no-implicit-this': { allow: ['isGlossaryOpen'] }.  no-implicit-this
  41:0  error  Unexpected {{log}} usage.  no-log

✖ 10 problems (10 errors, 0 warnings)

```

**Expectation:**
Only the valid errors(not todos) should show up in the output always;

```41:0  error  Unexpected {{log}} usage.  no-log```

_P.S_
I accidentally ran into this problem when I was integrating template linting as part of my git precommit workflow using lintstaged.

I'll be happy to give more details if needed. Thanks.

**Comment by scalvert** [comment]
Thanks for reporting, @shankarsridhar. This is a problem in the underlying `@lint-todo/utils` library. I've fixed it there, and we'll roll the fix in once it's released.

**Comment by bmish** [comment]
@scalvert I'd like to release a beta tomorrow, would this fix be ready for that? Note it could go in another beta if needed.
